### PR TITLE
ACS-3252 Refactor rule sets to handle ownership, inheritance and linking.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: src/main/webapp/definitions/alfresco-core.yaml [hunk 1]
```diff
@@ -11078,23 +11107,23 @@ definitions:
...
+      isAsynchronous:
         description: Whether the rule should be run in the background
         type: boolean
         default: false
       errorScript:
         description: If the rule should be run in the background then an optional error script can be referenced
         type: string
-      shared:
-        description: Whether the rule has been shared with more than one folder
+      isShared:
```
Comment (gevagel) [inline src/main/webapp/definitions/alfresco-core.yaml:11125]: Am I missing something or was this not declared with the other fields above (line 11097)?

Comment (tpage-alfresco) [reply src/main/webapp/definitions/alfresco-core.yaml:11125]: Line 11097 is part of:
```
    required:
      - id
      - name
      - actions
      - isEnabled
      - isInheritable
      - isAsynchronous
```

It controls which fields are mandatory (in the response)

Comment (gevagel) [reply src/main/webapp/definitions/alfresco-core.yaml:11125]: Thanks for the clarification Tom

**Review by gevagel — APPROVED** [review]
LGTM, also thanks for the quick demo earlier Tom

**Pull request merged**